Makefile

```diff
@@ -76,8 +79,15 @@ update-manifests:
 
 test: test-style test-unit
 
-test-cover:
-	${DEV_ENV_CMD} test-cover.sh
+test-cover: start-test-redis
+	docker run ${DEV_ENV_OPTS} \
+		-it \
+		--link ${REDIS_CONTAINER_NAME}:TEST_REDIS \
+		${DEV_ENV_IMAGE} bash -c 'DEIS_LOGGER_REDIS_SERVICE_HOST=$$TEST_REDIS_PORT_6379_TCP_ADDR \
+		DEIS_LOGGER_REDIS_SERVICE_PORT=$$TEST_REDIS_PORT_6379_TCP_PORT \
+		test-cover.sh' \
+		|| (make stop-test-redis && false)
+	make stop-test-redis
```

Comment:
why do you run `make stop-test-redis && false` above and then the same command here?